### DISCUSSION TITLE
Suggestion: Add X viewers instead of adding a maximum of 3 by 3

### DISCUSSION THREAD

**Discussion by wilsonricardopereirasilveira** [KevinBytesTheDust/CVAmp#134] [General]
Create a new field and instead of adding every 3 spectators, we have a field where we will add, for example, 10 spectators, and with that, it will start from 1 to 1, when it turns green and the instance is correct, it starts the next one, until the 10 spectators arrive, and if any instance dies, it automatically adds another instance, keeping 10 active instances, 10 is just an example, we can put a number inside the proxy number we have. This will save time of adding many instances and waiting to see if they all go up, with this, we put a number, and we know that we can go do something else, because when we come back, we will have that amount of instances watching.

We will also have the guarantee that this will be done in a calm and sequential way, guaranteeing that a high number of spectators will rise absolutely out of nowhere.

**Comment by SRZIsHere** [comment]
Good idea

**Comment by jlplenio** [comment]
I still like the current implementation because it demands user interaction and attention. This hinders automation and commercialization.

**Reply by SRZIsHere** [reply]
Maybe instead Split in half and push it to let says 10 max and where we could chose any number from 1 to 10 then ?
